### PR TITLE
readme: update badges

### PR DESCRIPTION
**Pull request by LeonidVas**
Replace Travis-CI with GHA badges.

### CODE REVIEW

**Review by oleg-jukovec — CHANGES_REQUESTED** [review]
Now when we click on the badge image, we get the image itself. It may be more useful. 

Before (click it):
fast_testing

For example, a link to workflow runs (click it):
[testing](https://github.com/tarantool/queue/actions/workflows/fast_testing.yml)

https://stackoverflow.com/a/61442205

File: README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
    <img src="https://avatars2.githubusercontent.com/u/2344919?v=2&s=250"
 align="right">
 </a>
-<a href="https://travis-ci.org/tarantool/queue">
-   <img src="https://travis-ci.org/tarantool/queue.png?branch=master"
-align="right">
-</a>
+
+![fast_testing](https://github.com/tarantool/queue/actions/workflows/fast_testing.yml/badge.svg?label=test)
```
[the remark this answers is not in the document]
Comment (LeonidVas) [reply README.md]: Yep. This was left after my "How it work?" research. Removed.

**Comment by LeonidVas** [comment]
> Now when we click on the badge image, we get the image itself. It may be more useful.
> 
> Before (click it): fast_testing
> 
> For example, a link to workflow runs (click it): [testing](https://github.com/tarantool/queue/actions/workflows/fast_testing.yml)
> 
> https://stackoverflow.com/a/61442205

OMG! Yep, I was wrong.
Updated.

**Review by oleg-jukovec — APPROVED** [review]

**Comment by LeonidVas** [comment]
Hi! Thank yo for the review. The PR has been updated.

**Review by oleg-jukovec — APPROVED** [review]
Thank you for the patch! LGTM.

**Pull request merged**